godel_robots/abb/godel_irb2400/godel_irb2400_support/launch/irb2400_blending_client.launch

```diff
@@ -9,16 +9,22 @@
   <arg name="robot_model_plugin" default="abb_irb2400_descartes/AbbIrb2400RobotModel"/>
 
   <group if="$(arg sim_sensor)" ns="$(arg namespace)">
+    <!-- simulated sensor -->
     <remap from="generated_cloud" to="sensor_point_cloud"/>
     <node pkg="godel_surface_detection" type="point_cloud_generator_node" name="point_cloud_generator_node">
       <rosparam command="load" file="$(find godel_irb2400_support)/config/point_cloud_descriptions.yaml"/>
     </node>
   </group >
 
+  <group if="$(arg sim_robot)" ns="$(arg namespace)">
+    <!-- industrial_robot_simulator: accepts robot commands and reports status -->
```

Comment:
The **accepts robot commands and reports status**  line could be slightly misleading, I think the name of the node itself is self-explanatory.